### ISSUE TITLE
README editing embedded in files app not accessible with keyboard / controls invisible until focussed

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#2381]
- [x] Menubar is only visible then the editor has focus, which makes it hard to discover
- [x] When trying to navigate with the keyboard the menu bar should stay visible and be navigable with the keyboard
- [x] Placeholder is not accessible by keyboard navigation (to create a new README)
